src/onnxruntime_benchmark/CMakeLists.txt

```
@@ -30,6 +29,15 @@ add_executable(${PROJECT_NAME} ${sources} ${headers})
 
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/include" ${ORT_INCLUDE})
 
-target_link_libraries(${PROJECT_NAME} PRIVATE gflags nlohmann_json::nlohmann_json ${OpenCV_LIBRARIES} ${ORT_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} PRIVATE gflags ${OpenCV_LIBRARIES} ${ORT_LIBRARIES})
+
+# search for nlohmann-json
+find_package(nlohmann_json QUIET HINTS /usr/lib/cmake)
```

```suggestion
find_package(nlohmann_json REQUIRED HINTS /usr/lib/cmake)
```
если без данного пакета сборка невозможна, то он должен быть REQUIRED, и поиск надо делать в начале cmake файла (6 строка, после поиска opencv).